cpu/cortexm_common/Makefile

```diff
@@ -2,6 +2,9 @@
 # optimization. see #5774.
 SRC_NOLTO += thread_arch.c
 
+# see #5775
```

Comment:
I know it just says `#5774` above, but could you put an URL here instead?
https://github.com/RIOT-OS/RIOT/issue/XXX or something

Reply:
Good idea, I've amended the URL for both issues.